js/index.js

```diff
@@ -37,6 +37,103 @@ const siteContent = {
   },
 };
 
+///////Beginning of variables
+const nav = document.querySelector('nav');
+const navItems = Array.from(nav.children);
+const headerImg = document.querySelector('header img');
```

Comment:
For your variable assignments, the directions actually asked for the querySelector invocations for images to use ID selectors (as opposed to class selectors).